cvat-canvas/src/typescript/editHandler.ts

```diff
@@ -383,6 +385,18 @@ export class EditHandlerImpl implements EditHandler {
         this.release();
     }
 
+    private onNKeyUp = (e: KeyboardEvent): void => {
```

Comment:
It must be handled on `cvat-ui` level with mousetrap. 
Otherwise it cannot be configured.

- When shortcut caught in cvat-ui, it should call `canvasInstance.edit({enabled: false})`. 
- Canvas should handle correctly editing finish and trigger `canvas.edited` event. 

Please, see how it works for masks in this file:  cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/controls-side-bar.tsx 

![image](https://github.com/cvat-ai/cvat/assets/40690378/075e4faf-ffcb-49d5-be29-8fbe5ac77b19)

Try to edit mask and press N after finish editing.